lib/files.gd

```diff
@@ -316,6 +316,63 @@ DeclareOperation( "ReadTest", [ IsString ] );
 ##
 DeclareOperation( "ReadAsFunction", [ IsString ] );
 
+#############################################################################
+##
+#O  ReadAsExpression( <name-file> ) . . . . . . . . read a file as expression
+##
+##  <#GAPDoc Label="ReadAsExpression">
+##  <ManSection>
+##  <Oper Name="ReadAsExpression" Arg='name-file'/>
+##
+##  <Description>
+##  reads the file with filename <A>name-file</A> as a GAP expression.
```

Comment:
I would not use 'name-file' here because for GAP this means "name minus file", how about 'fname' instead?